db/schema.sql

```diff
@@ -47,16 +48,26 @@ CREATE TABLE IF NOT EXISTS user_trip (
   UNIQUE (user_email, trip_id)
 );
 
-CREATE TABLE IF NOT EXISTS trip_destination (
+CREATE TABLE IF NOT EXISTS trip_destination_poi (
```

Comment:
it looks like this table was changed to be the exact same as the one under it? let me know if this was intended